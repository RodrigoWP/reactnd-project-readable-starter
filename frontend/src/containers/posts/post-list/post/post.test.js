'use strict'

import React from 'react'
import { shallow } from 'enzyme'
import { Post } from '.'

it('Should render correctly', () => {
  const component = (
    <Post
      data={{
        id: '0',
        category: 'category'
      }}
      deletePost={jest.fn}
      editPost={jest.fn}
    />
  )

  expect(shallow(component)).toMatchSnapshot()
})
