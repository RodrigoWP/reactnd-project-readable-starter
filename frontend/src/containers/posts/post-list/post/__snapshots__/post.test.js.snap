// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div>
  <CategoryTag
    category="category"
  />
  <CrudMenu
    handleEdit={[Function]}
    handleRemove={[Function]}
    id="0"
  />
  <div>
    <span />
  </div>
  <div>
    <span />
  </div>
  <div>
    <div>
      <span />
      <span>
        Invalid date
      </span>
    </div>
    <div>
      <PostScore
        commentScore={0}
        onClickVoteDown={[Function]}
        onClickVoteUp={[Function]}
      />
    </div>
    <Button
      height="40px"
      icon=""
      onClick={[Function]}
      width="40px"
    />
  </div>
</div>
`;
